### PR TITLE
replace underscore with standalone lodash modules

### PR DESCRIPTION
**Pull request by dhritzkiv**
Also, updated devDependencies (for up-to-date testing)

### CODE REVIEW

**Comment by latentflip** [comment]
Awesome. I'm not clear though whether we're supposed to be using `require('lodash.assign')` or `require('lodash/object/assign')`. One being an actual separate module, the other just being just the files we need from the full lodash. I _think_ the latter but I'm not sure. @HenrikJoreteg?

**Comment by latentflip** [comment]
Derp, my bad, I read the post again and see we're doing lodash.*. Ignore me!

File: ampersand-collection-view.js [hunk 1]
```diff
@@ -25,23 +29,24 @@ function CollectionView(spec) {
...
     },
     _getViewByModel: function (model) {
-        return _.find(this.views, function (view) {
+        return find(this.views, function (view) {
             return model === view.model;
         });
     },
     _createViewForModel: function (model, renderOpts) {
-        var view = new this.view(_({model: model, collection: this.collection}).extend(this.viewOptions));
+	    var defaultViewOptions = {model: model, collection: this.collection};
```
Comment (latentflip) [inline ampersand-collection-view.js]: Seems we've gained a couple of extra indent spaces here? At least according to the github ui.

**Comment by latentflip** [comment]
Apart from the comment above, this looks good to me visually.

**Comment by wraithgar** [comment]
+1 looks good once the whitespacing is ironed out

**Comment by dhritzkiv** [comment]
It was purely visual; I had a tab character (personal editor preference) which is equivalent to 4 spaces, but Github's UI shows it as 8 spaces for some reason. Fixed anyway.

In the future, will using tabs be acceptable in commits to @AmpersandJS modules?

**Comment by latentflip** [comment]
@dhritzkiv no, spaces only please :)

**Comment by wraithgar** [comment]
Released as `v1.3.01`

Meant to do a patch-release, but accidentally typed 'minor'

**Pull request merged**